scripts/ensemble_obspace_diag.py

```diff
@@ -0,0 +1,320 @@
+import emcpy.utils.utils as utils
```

Comment:
I don't think this belongs in `emcpy`.

Reply:
I like the idea of there being some sort of 'gallery' though of useful examples for people to see how things are used, the 'examples' directory is a bit more basic whereas this would be more practical.

Reply:
What about putting this script in to GSI-utils? I have priviliges there to review/merge and it might make more sense there since it is using GSI ncdiags

Reply:
@CoryMartin-NOAA I've moved this out of emcpy. I will work on getting it into GSI-utils as you have suggested.